WTWSMS/history/characters/WtWSMS_erikw_characters.txt

```diff
@@ -8006,7 +8008,7 @@
 	religion = hellenic_pagan
 	culture = romano_hispanic
 	dynasty = 1022225
-	mother = 1101003
+	mother = 1101002
```

Comment:
Seems OK and 1101003 doesn't, could you recheck @loup99?

Reply:
Yes.